src/components/ProductList/index.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react'
+import Link from 'next/link'
 
 const ProductList = () => {
   return (
-      <section className="product-list">
-        <header>
-          <h2 className="product-list__title">Product List</h2>
-          <p className="product-list__description">
-            This is a sample description
-          </p>
-        </header>
-      </section>
+    <section className="product-list">
+      <header>
+        <h2 className="product-list__title">
+          <Link href="/product-detail">
+            <a className="product-list__link-to-detail">Product List</a>
+          </Link>
+        </h2>
+        <p className="product-list__description">
+          This is a sample description
+        </p>
+      </header>
+    </section>
```

Comment:
I thought you were going to create a `1 x 3` `product-article`. I think it's better than creating `&__link-to-detail` class which it isn't relate to the `product-article` at all 🤔

Reply:
First, I thought just to make it clickable in the first place as we focus on seeing the product detail screen. I can re-design it to use `article` tag instead in the next implement for the product list page.

Reply:
Just a simple three `div` box should do, since I don't want to create a css class `&__link` for `product-list`.

Reply:
Alright.